jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet

```diff
@@ -1,214 +1,217 @@
 {
-  _config+:: {
-    namespace: 'default',
-    versions+:: { nodeExporter: 'v1.0.1' },
-    imageRepos+:: { nodeExporter: 'quay.io/prometheus/node-exporter' },
+  nodeExporter+: {
+    _config+:: {
+      namespace: $._config.namespace,  //FIXME(paulfantom): figure out how to pass this global config in a better way
```

Comment:
I like to use the `error "must provide namespace"` approach, which fails to generate if the config is not set, and forces the caller to configure it.